### PR TITLE
A bunch of coverage improvements (re for models in inference snext50, BERT_pytorch, mobilenet_v3_large, pytorch_CycleGAN_and_pix2pix, dcgan, resnet18, mnasnet1_0)

### PR DESCRIPTION
**Pull request by Chillee**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #86067
* __->__ #86050
* #86021
* #85998

### CODE REVIEW

File: functorch/_src/aot_autograd.py [hunk 1]
```diff
@@ -275,6 +275,7 @@ class AOTConfig:
 
 def aot_dispatch_base(flat_fn, flat_args: List[Tensor], aot_config: AOTConfig):
     fw_module = make_fx(flat_fn, aot_config.decompositions)(*flat_args)
+    breakpoint()
```
Comment (ezyang) [inline functorch/_src/aot_autograd.py:278]: Remove

File: torch/_meta_registrations.py [hunk 2]
```diff
@@ -105,6 +105,20 @@ def meta_index_select_out(self, dim, index, out):
...
+        aten.relu_.default,
+        aten.add_.Tensor,
+        aten.hardtanh_.default,
+        aten.hardswish_.default,
+        aten.leaky_relu_.default,
+    ],
+    register_dispatcher=False,
+)
+def meta_inplace_(self, *args, **kwargs):
+    return self
```
Comment (ezyang) [inline torch/_meta_registrations.py:119]: This is incomplete, not enough error checking

**Review by ezyang — APPROVED** [review]

File: torch/_meta_registrations.py (hunk 2 above)
Comment (Chillee) [reply torch/_meta_registrations.py:119]: uh... I guess this is fine for unary ops? Or I suppose unary ops need to check dtypes as well. Annoying.

Comment (Chillee) [reply torch/_meta_registrations.py:119]: I guess we should just make full decompositions for some of these lol.

Comment (ezyang) [reply torch/_meta_registrations.py:119]: Yeah, dtypes. Ofc I know our error checking is woefully inadequate, will have to be addressed differently...

**Pull request closed without merging**